script/testing/jdbc/src/PelotonTest.java

```diff
@@ -14,7 +14,7 @@
 import org.postgresql.util.*;
 
 public class PelotonTest {
-  private final String url = "jdbc:postgresql://localhost:54321/";
+  private final String url = "jdbc:postgresql://localhost:5432/";
```

Comment:
I usually shutdown Postgres when I run Peloton. I guess we should create a new issue to fix a non-colliding port number for Peloton, once and for all.